### ISSUE TITLE
Feature space is created in LoadConnections

### ISSUE THREAD

**Issue by nicola** [yala/learning_interactive_planning#32]
Avoid feature space being created in LoadConnection,
do a first pass on the all the text_features, and second pass to load the ones we got in answers

**Comment by nicola** [comment]
```
bool FeatureSpace::LoadFeatureMapping (String _sName)
bool FeatureSpace::SaveFeatureMapping (String _sName)
```

Instead of having hacks we could just use these two functions, and store the feature space into a file and just load it

**Comment by nicola** [comment]
solved in #30
